test/unit/foreman/renderer/renderers_shared_tests.rb

```diff
@@ -152,7 +152,11 @@ module RenderersSharedTests
 
     test "should find all registered host statuses" do
       source = OpenStruct.new(content: '<%= all_host_statuses.map { |s| s.status_name }.join(",") %>')
-      assert_equal(renderer.render(source, @scope), "Build,Configuration")
+      statuses = renderer.render(source, @scope).split(',')
+
+      refute_nil statuses.index('Build')
```

Comment:
assert_includes gives a better error message if this breaks.

Reply:
Good point